packages/tools/devtools/devtools-browser-extension/src/devtools/TelemetryLogging.ts

```diff
@@ -181,6 +181,7 @@ export class OneDSLogger implements ITelemetryBaseLogger {
 				["Data.extensionVersion"]: extensionVersion,
 				["Data.sessionID"]: this.sessionID,
 				["Data.continuityID"]: this.continuityID,
+				["Data.category"]: category,
```

Comment:
I think I lean towards removing this line and the whole computation of `category` above. It may be that right now all the events for Devtools don't have one and are getting "Generic" because of line 171... and in that case I think it's better to just start by not having the column/property at all. If we start giving an explicit category to events, the logic in lines 189-203 already ensures we'll get the value in the table.